security-team/security-groups.md

```diff
@@ -25,3 +25,4 @@ Members:
 
 - [wangzefeng@huawei.com](mailto:wangzefeng@huawei.com)
 - [renhongcai@huawei.com](mailto:renhongcai@huawei.com)
+- [guyue0864@gmail.com](mailto:guyue0864@gmail.com)
```

Comment:
We can also add the member's name and GitHub ID if we add personal emails. some like:
```
Hongcai Ren([@RainbowMango](https://github.com/rainbowmango)), [renhongcai@huawei.com](mailto:renhongcai@huawei.com)
```

Reply:
Good idea, it does look better